front/src/applications/operationalStudies/Home.tsx

```diff
@@ -125,14 +99,52 @@ export default function Home() {
     setSortOption(e.target.value as SortOptions);
   };
 
+  const toggleProjectSelection = (id?: number) => {
+    if (id !== undefined) {
```

Comment:
perhaps use lodash `isNil` to avoid having this comparison in many places

```suggestion
    if (!isNil(id)) {
```

Reply:
Just in case, I'm pretty sure you don't even need all that. `if (id) {` tests if id is defined.

Reply:
Mmm I prefer to use built in simple comparison function with `undefined` rather than call an extern function from a lib, with no real benefit, no ?

Reply:
For real though, you could just put `if (id)`, no need to even compare it to undefined ;)

Reply:
`if (id)` will fail if id is `0` or `''`

Reply:
Mb, cheers for that mate :)

Reply:
Exactly, because `id` could be 0 and by consequence return... false !